src/zenml/integrations/github/code_repositories/github_code_repository.py

```diff
@@ -202,3 +215,55 @@ def check_remote_url(self, url: str) -> bool:
             return True
 
         return False
+
+
+def create_symlink_in_local_repo_copy(
+    symlink_source: str, symlink_target: str
+) -> None:
+    """This function attempts to create a symbolic link at `local_path` that points to `symlink_target`.
+
+    If a file or directory already exists at `local_path`, it will
```

Comment:
```suggestion
    If a file or directory already exists at `symlink_source`, it will
```

Reply:
I have renamed the attributes to `symlink_dst` and `symlink_src`, corresponding to the attribute name of `os.path.symlink`.